domains/auth/Auth.API/Tests/Resources/AddEnvironmentVariables.cs

```diff
@@ -0,0 +1,18 @@
+using System;
```

Comment:
When testing with environment variables have a look at how to ensure they are cleaned up at:  https://github.com/Energinet-DataHub/eo-data-sync/blob/main/services/DataSync.API/Tests/Helper/EnvironmentBase.cs

and here for seriel execution to ensure that unittests with different variables don't accidentially share environment variables.   [  [Collection("environment-tests")]](https://github.com/Energinet-DataHub/eo-data-sync/blob/c8393d97708615581a97a7138a6355b22a79e1d7/services/DataSync.API/Tests/Helper/ConfigurationTests.cs#L8)